shap/plots/dependence.py

```diff
@@ -138,6 +142,15 @@ def convert_name(ind):
     if categorical_interaction and clow != chigh:
         bounds = np.linspace(clow, chigh, chigh - clow + 2)
         color_norm = matplotlib.colors.BoundaryNorm(bounds, colors.red_blue.N)
+        
+    # optionally add jitter to feature values
+    if x_jitter > 0:
+        if x_jitter > 1: x_jitter = 1
+        xvals = list(set(xv))
+        smallest_diff = np.min(np.diff(np.sort(xvals)))
+        jitter_amount = x_jitter * smallest_diff
+        for i in range(len(xv)):
+            xv[i] += (np.random.ranf(size=1)[0] * jitter_amount) - (jitter_amount/2)
```

Comment:
```suggestion

```
Here is the deletion of the line previously in the for loop. For future reference you can just change the code on your machine and then commit/push to GH with the changes and the Pull Request will automatically update.